### PR TITLE
Add some demos for new query pipeline

### PR DESCRIPTION
**Pull request by smitpatel**

### CODE REVIEW

**Comment by smitpatel** [comment]
@divega - This is ready for review.

File: NQueries/Program.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                    });
+
+                context.SaveChanges();
+            }
+
+            using (var context = new BloggingContext())
+            {
+                var cities = context.Cities.Where(c => c.Country == "USA").ToList();
+
+                var query = context.People
```
Comment (divega) [inline NQueries/Program.cs]: Maybe query comprehension syntax to make it easier to read for everyone?

File: NQueries/Program.cs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                    {
+                        FirstName = "Maurycy",
+                        LastName = "Markowsky",
+                        City = "Seattle"
+                    },
+                    new Person
+                    {
+                        FirstName = "Shay",
+                        LastName = "Rojansky",
+                        City = "Berlin"
```
Comment (divega) [inline NQueries/Program.cs:44]: Add yourself for the full snowshoeing team :smile:

File: NuGet.config [hunk 3]
```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="Custom" value="C:\Users\smpatel\Desktop\Shipping\" />
```
Comment (divega) [inline NuGet.config]: Maybe you want to avoid checking this one it?

**Review by divega — APPROVED** [review]
LGTM. Only a few comments.

**Pull request merged**